### ISSUE TITLE
Crash on N1 reload

### ISSUE THREAD

**Issue by mbilker** [nylas/N1#450]
Every so often when I reload the page via Developer --> Reload, the N1 window closes and doesn't show back up. Checking `htop` shortly after shows the N1 process still running, but exits around 5 seconds after the window close. If I were to execute the program for N1 (`/usr/share/nylas/nylas` in my case) before the program exits, a partially initialized N1 window opens where the main window is unresponsive, but the menu buttons work and the Developer Tools window works.

I have not tested if the Developer Tools Elements tab is view-able. When I checked all the N1 logs for that process, they don't show any information pertaining to the class.

If I encounter the issue again I will take screenshots and post log files.

**Comment by bengotow** [comment]
Hey @mbilker — I think I've run into this a couple times. If you can pull together logs and other info, we can probably narrow it down. I have a feeling it might be an Electron issue, but I'm not 100% sure.

**Comment by mbilker** [comment]
Ok. I found a similar issue with a recent version. Went to reload after changing a LESS stylesheet for mbilker/email-pgp, but this showed up and none of the menu buttons worked. Quit N1 afterwards. I know this issue is more towards development and not general user usage.

![2015-11-25-171016_1280x800_scrot](https://cloud.githubusercontent.com/assets/1266541/11410748/ff7bf61c-9397-11e5-8b6b-7426e1dd361e.png)

Logs: https://dl.dropboxusercontent.com/u/425945/Nylas-27417.zip
I noticed a deprecation notice in Nylas-N1-27417.log saying a restarts are no longer supported.
